crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -57,8 +58,11 @@ impl<'a, W> TerminalPane<'a, W> {
         };
 
         match self.section {
-            LayoutSections::Pane => build_message_vec(FOOTER_TEXT_ACTIVE),
-            LayoutSections::TaskList => build_message_vec(FOOTER_TEXT_INACTIVE),
+            LayoutSections::Pane => build_message_vec(EXIT_INTERACTIVE_HINT),
+            LayoutSections::TaskList => {
+                // Spaces are used to pad the footer text for aesthetics
+                build_message_vec(format!("{}   {}", ENTER_INTERACTIVE_HINT, SCROLL_LOGS).as_str())
+            }
```

Comment:
Up to you, but you can dedupe the spacing logic so it's only contained in `build_message_vec`:
```
let build_message_vec = |footer_texts: &[&str]| -> Line {
    let mut messages = Vec::new();
    messages.extend_from_slice(footer_texts);
```

```suggestion
            LayoutSections::Pane => build_message_vec(&[EXIT_INTERACTIVE_HINT]),
            LayoutSections::TaskList => {
                // Spaces are used to pad the footer text for aesthetics
                build_message_vec(&[ENTER_INTERACTIVE_HINT, SCROLL_LOGS])
            }
```

Reply:
Some of my best "past midnight with a screaming baby" code, truly.